mds/apis/agency_api/v0_x/compliance.py

```diff
@@ -0,0 +1,121 @@
+from rest_framework import serializers
+from rest_framework import viewsets
+
+from django.db.models import Q, Prefetch
+
+from mds import models
+
+from datetime import datetime
+import pytz
+
+
+class ComplianceSerializer(serializers.ModelSerializer):
+    rules_id = serializers.SerializerMethodField()
+
+    compliances = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.Policy
+        fields = ("id", "rules_id", "compliances")
+
+    def get_rules_id(self, policy):
+        rules = []
+        for rule in policy.rules:
+            rules.append(rule["rule_id"])
+        return rules
+
+    def get_compliances(self, policy):
```

Comment:
I can't see where this is called from. 
Maybe some code that you don't need anymore ?

Reply:
Oh, compliance isn't a real attribute of Policy... :+1: